src/main/webapp/app/exercises/quiz/manage/re-evaluate/multiple-choice-question/re-evaluate-multiple-choice-question.component.ts

```diff
@@ -34,11 +34,7 @@ export class ReEvaluateMultipleChoiceQuestionComponent {
     @Input()
     backupQuestion: MultipleChoiceQuestion;
 
-    /**
-     * Constructs the re-evaluate component.
-     * @param artemisMarkdown the ArtemisMarkdownService
-     */
-    constructor(private artemisMarkdown: ArtemisMarkdownService) {}
+    constructor() {}
```

Comment:
According to https://stackoverflow.com/questions/63486890/angular-do-we-need-a-constructor-if-it-doesnt-have-any-logic the empty constructor is not needed

Reply:
good point. I remove it